lib/index.mjs

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import stringify from './stringify.js';
+import parse from './parse.js';
+import formats from './formats.js';
+
+export default {
```

Comment:
there's no value or point in the default export if you have the names :-)

Reply:
I was thinking that people could do this, like they currently can with the help of bundlers:

```js
import qs from 'qs';

qs.stringify('...');
```

And I think this PR might actually not be a breaking change because of this.